lib/java_buildpack/container/tomcat/tomcat_geode_store.rb

```diff
@@ -44,8 +44,10 @@ def release
         credentials = @application.services.find_service(FILTER, KEY_LOCATORS, KEY_USERS)['credentials']
         user = credentials[KEY_USERS].find { |u| cluster_operator?(u) }
 
-        @droplet.java_opts.add_system_property 'gemfire.security-username', user['username']
-        @droplet.java_opts.add_system_property 'gemfire.security-password', user['password']
+        File.open(@droplet.root + 'gemfire.properties', 'w') do |f|
```

Comment:
Do we ever need to worry about merging with an existing `gemfire.properties` file. E.g do you imagine users would ever configure other features of their app with a `gemfire.properties` in the app root?

Reply:
This is a good point.  It's something Ben and I discussed in the process, but have not revisited.  I will do some experimentation.

Reply:
Maybe using `gfsecurity.properties` instead of `gemfire.properties` mitigates the risk here? Users really shouldn't be pushing a sensitive credentials in their app.

Reply:
Deleted the addition of this file altogether in favor of pulling the credentials directly from `VCAP_SERVICES` in our ClientAuth class.